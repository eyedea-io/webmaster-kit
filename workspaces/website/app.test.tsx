import * as React from 'react'
import * as ReactShallowRenderer from 'react-test-renderer/shallow'
import App from './app'

describe('<App />', () => {
  const renderer = ReactShallowRenderer.createRenderer()

  it('renders', () => {
    expect(renderer.render(<App />)).toMatchSnapshot()
  })
})
