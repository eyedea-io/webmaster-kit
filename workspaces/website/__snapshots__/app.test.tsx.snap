// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders 1`] = `
<SentryErrorBoundary>
  <Provider
    store={
      Object {
        "formStore": Object {
          "forms": Array [],
        },
        "langStore": Object {
          "language": "EN",
        },
        "modal": Object {
          "active": undefined,
        },
        "userStore": Object {
          "profile": undefined,
          "token": "",
        },
      }
    }
  >
    <ThemeProvider
      theme={
        Object {
          "colors": Object {
            "default": Object {
              "hex": "#56687C",
              "hsl": "211, 18%, 41%",
              "hsla": [Function],
              "rgb": "86, 104, 124",
              "rgba": [Function],
            },
            "icon": Object {
              "hex": "#ccc",
              "hsl": "236, 100%, 40%",
              "hsla": [Function],
              "rgb": "0, 12, 204",
              "rgba": [Function],
            },
            "meta": Object {
              "hex": "#999999",
              "hsl": "0, 0%, 60%",
              "hsla": [Function],
              "rgb": "153, 153, 153",
              "rgba": [Function],
            },
            "negative": Object {
              "hex": "#db2828",
              "hsl": "0, 71%, 50%",
              "hsla": [Function],
              "rgb": "219, 40, 40",
              "rgba": [Function],
            },
            "positive": Object {
              "hex": "#26c344",
              "hsl": "131, 67%, 45%",
              "hsla": [Function],
              "rgb": "38, 195, 68",
              "rgba": [Function],
            },
            "primary": Object {
              "hex": "#494de4",
              "hsl": "238, 74%, 59%",
              "hsla": [Function],
              "rgb": "73, 77, 228",
              "rgba": [Function],
            },
            "ui": Object {
              "hex": "#d3d7e0",
              "hsl": "221, 17%, 85%",
              "hsla": [Function],
              "rgb": "211, 215, 224",
              "rgba": [Function],
            },
          },
          "fontFamilies": Object {
            "athelas": "athelas, georgia, serif",
            "avenir": "'avenir next', avenir, sans-serif",
            "baskerville": "baskerville , serif",
            "bodoni": "\\"Bodoni MT\\", serif",
            "calisto": "\\"Calisto MT\\", serif",
            "code": "Consolas, monaco, monospace",
            "courier": "'Courier Next', courier, monospace",
            "garamond": "garamond, serif",
            "georgia": "georgia, serif",
            "helvetica": "'helvetica neue', helvetica, sans-serif",
            "sansSerif": "-apple-system, BlinkMacSystemFont, 'avenir next', avenir,
      'helvetica neue', helvetica, ubuntu, roboto, noto, 'segoe ui', arial, sans-serif
    ",
            "serif": "georgia, times, serif",
            "times": "times, serif",
          },
          "fontSizes": Object {
            "alpha": "32px",
            "beta": "28px",
            "delta": "20px",
            "epsilon": "18px",
            "gamma": "24px",
            "giga": "64px",
            "kilo": "40px",
            "mega": "48px",
            "micro": "13px",
            "milli": "14px",
            "nano": "12px",
            "zeta": "16px",
          },
          "letterSpacing": Object {
            "mega": ".25em",
            "tight": ".05em",
            "tracked": ".1em",
          },
          "lineHeights": Object {
            "copy": 1.5,
            "solid": 1,
            "title": 1.25,
          },
          "media": Object {
            "desktop": [Function],
            "phone": [Function],
            "tablet": [Function],
            "tabletAndDown": [Function],
            "tabletAndUp": [Function],
          },
          "radius": "5px",
          "spacing": Object {
            "lg": "48px",
            "md": "32px",
            "sm": "24px",
            "toString": [Function],
            "xl": "64px",
            "xs": "16px",
            "xxl": "96px",
            "xxs": "8px",
            "xxxl": "128px",
            "xxxs": "4px",
          },
        }
      }
    >
      <React.Fragment>
        <inject-Routes-with-store />
        <Unknown />
        <GlobalStyleComponent />
        <GlobalStyleComponent />
      </React.Fragment>
    </ThemeProvider>
  </Provider>
</SentryErrorBoundary>
`;
